src/main/java/com/connexta/search/query/configs/QueryManagerConfiguration.java

```diff
@@ -50,21 +49,16 @@ public DataStore dataStore(@NotNull final URL solrUrl) throws IOException {
     final SolrLayerConfiguration solrLayerConfiguration =
         new SolrLayerConfiguration(new ArrayList<>());
     solrLayerConfiguration.setLayerName(SolrConfiguration.LAYER_NAME);
-    final List<SolrAttribute> layerAttributes = new ArrayList<>();
 
-    final SolrAttribute contentsSolrAttribute =
-        new SolrAttribute(SolrConfiguration.CONTENTS_ATTRIBUTE_NAME, String.class);
-    contentsSolrAttribute.setEmpty(false);
-    contentsSolrAttribute.setUse(true);
-    layerAttributes.add(contentsSolrAttribute);
+    SolrConfiguration.QUERY_TERMS.stream()
+        .forEach(
+            s -> {
```

Comment:
✏️ Change `s` to `term` for clarity.

Reply:
In some places we refer to them as "terms" (e.g. searchTerms Solr core). In other places we refer to them as "attributes" (e.g. `CONTENTS_ATTRIBUTE_NAME`). Do "term" and "attribute name" mean the same thing? Or do they mean different things?

Reply:
Good point, they are "fields" to Solr. Since we're configuring Solr here, it probably makes sense to use the same nomenclature. I don't think OGC CommonQL refers to them as "fields" when creating filters. I think they either refer to them as attributes or properties and that might have caused my confusion.

Reply:
Most (all?) databases also refer to them as "fields". For this PR, I'll change the local var name to "term" like you suggested. If we want  to standardize our language at point in the future we can. For now, I'll think of "term", "field", "attribute", and "property" as interchangeable.